index.php

```diff
@@ -3,10 +3,42 @@
 require_once __DIR__.'/vendor/autoload.php';
 
 use Heise\Shariff\Backend;
-use Zend\Config\Reader\Json;
 
+/**
+ * Demo Application using Shariff Backend
+ */
 class Application
 {
+    /**
+     * Sample configuration
+     *
+     * Format for this example is JSON for nice readability
+     *
+     * @var string
+     */
+    private static $configuration = '
```

Comment:
Why not use a PHP data structure directly? Is there a point to parsing a json string into a data structure?